genesyscloud/routing_utilization/resource_routing_utilization_utils.go

```diff
@@ -68,15 +129,16 @@ func FlattenUtilizationSetting(settings MediaUtilization) []interface{} {
 	return []interface{}{settingsMap}
 }
 
-func FilterAndFlattenLabelUtilizations(labelUtilizations map[string]LabelUtilization, originalLabelUtilizations []interface{}) []interface{} {
+// To be removed once resource_user is refactored to use FilterAndFlattenLabelUtilizations instead
```

Comment:
This is the small bit of duplication I mentioned, these Internal functions.

Like I said, I'll get rid of all of them in my next PR, just separated the work to keep each PR smaller and easier to review.